core/trino-main/src/main/java/io/trino/connector/system/jdbc/ColumnJdbcTable.java

```diff
@@ -19,12 +19,11 @@
 import io.airlift.slice.Slices;
 import io.trino.FullConnectorSession;
 import io.trino.Session;
+import io.trino.connector.informationschema.SystemTableFilter;
```

Comment:
> Change schema/table prefix behavior in system.jdbc and info_schema

Assuming "info_schema" is an abbreviation for information_schema, please spell it out.

Also, the commit seems not to touch information_schema.
So spell out "system.jdbc.columns"

Reply:
> Schema and table prefixes are treated equal when limiting their numbers
during column queries in system.jdbc and information_schema. But in
reality fetching tables from 100 schemas and then filtering them in
memory is way more expensive than fetching 100 tables directly.
>
> This is especially true if schemas have lots of tables underneath that
we will have to go through.

Thanks for adding some color to the commit message. This introduces the problem it is attempting to solve. Can the commit message also summarize the solution?